lib/operations/admin_ops.js

```diff
@@ -59,4 +62,10 @@ function validateCollection(admin, collectionName, options, callback) {
   });
 }
 
+/**
+ * @callback AdminResultCallback
+ * @param {Error} [error]
```

Comment:
I can add for this one, but there's actually 131 things missing documentation, I feel like it would make sense to make another ticket to fill them out, in this epic so its before 4.0 release, does that make sense?
regex for missing doc: `@\w+ \{\w+\} \w+$`